packages/block-library/src/navigation/index.php

```diff
@@ -429,6 +429,37 @@ function block_core_navigation_block_contains_core_navigation( $inner_blocks ) {
 	return false;
 }
 
+/**
+ * Create and returns a navigation menu containing a page-list as a fallback.
+ *
+ * @return array the newly created navigation menu.
+ */
+function block_core_navigation_get_default_pages_fallback() {
+	$registry = WP_Block_Type_Registry::get_instance();
+
+	// If `core/page-list` is not registered then use empty blocks.
+	$default_blocks = $registry->is_registered( 'core/page-list' ) ? '<!-- wp:page-list /-->' : '';
+
```

Comment:
This condition does not exist in the editor. Why?

Reply:
Good catch. I added a similar check to the editor.